src/app/personside/kontrollsporsmal/skjulPåTversAvVinduerUtils.ts

```diff
@@ -1,19 +1,36 @@
 import moment from 'moment';
 import * as Cookies from 'js-cookie';
 
-const cookieNavn = 'SkjulKontrollSpørsmålFor';
+const skjulForCookieNavn = 'SkjulKontrollSpørsmålFor';
+const skjulVedPlukkCookieNavn = 'SkjulKontrollSpørsmålJobberMedPlukk';
 
 export function settSkjulKontrollspørsmålPåTversAvVinduerForBrukerCookie(fnr: string) {
     const omEnTime = moment()
         .add(1, 'hour')
         .toDate();
-    Cookies.set(cookieNavn, fnr, { expires: omEnTime });
+    Cookies.set(skjulForCookieNavn, fnr, { expires: omEnTime });
 }
 
 export function kontrollspørsmålHarBlittLukketForBruker(fnr: string): boolean {
-    const cookie = Cookies.get(cookieNavn);
+    const cookie = Cookies.get(skjulForCookieNavn);
     if (cookie && cookie === fnr) {
         return true;
     }
     return false;
 }
+
+export function settJobberMedSpørsmålOgSvar() {
+    Cookies.set(skjulVedPlukkCookieNavn, 'true');
```

Comment:
Trenger denne også logikk for brukeren det gjelder? Uten helt å ha oversikt vet jeg ikke hvorfor den ene er brukerpesifikmt, mens den andre er browser

Reply:
Denne skal være "global". Den action du gjør, skal avgjøre om den neste brukeren du slår opp skal vise kontrollspørsmål eller ikke. Derfor skal den ikke vøre knyttet til bruker, men hendelse i applikasjonen.